Contrib/FreeWilson/freewilson.py

```diff
@@ -0,0 +1,486 @@
+# Copyright (C) Brian Kelley 2021
+#
+#   @@ All Rights Reserved @@
+#  This file is part of the RDKit.
+#  The contents are covered by the terms of the BSD license
+#  which is included in the file license.txt, found at the root
+#  of the RDKit source tree.
+#
+
+"""
+Free Wilson Analysis
+
+A Mathematical Contribution to Structurre-Activity Studies
+Spencer M, Free, Jr and James W Wilson
+Journal of Medicinal Chemistry
+Vol 7, Number 4, July 6, 1964
+
+Basic usage: get a scaffold (or scaffolds) some compounds and their scores, 
+then run freewilson:
+
+
+```
+>>> import sys
+>>> freewilson impor FWDecompose, FWBuild, predictions_to_csv
+>>> decomp = FWDecompose(scaffold, mols, scores)
+>>> preds = FWBuild(fdecomp, 
+...                    pred_filter=lambda x: x > 8, 
+...                    mw_filter=lambda mw: 100<mw<550)
+>>> predictions_to_csv(sys.stdout, preds)
+```
+
+Scores need to be in an appropriate form for regrerssion analysis, i.e. pIC50s as opposed to IC50s.
+Enumerations are a lot faster if you know a prediction cutoff or molecular weight cutoff.
+
+Scaffolds can be any scaffold or smarts pattern or list of either.  The system automatically 
+enumerates matching cores in the analysis.
+
+To see if the decomposition is useful, check the r squared value
+
+```
+>>> print(r"Training R^2 is {decomp.r2}")
```

Comment:
```suggestion
>>> print(f"Training R^2 is {decomp.r2}")
```

Reply:
This typo makes it look like the sample code in the doc strings hasn't been tested.
Maybe a good idea to do that.